jumpscale/servers/threebot/threebot.py

```diff
@@ -241,13 +234,21 @@ def list_all(self):
     def add(self, path: str = None, giturl: str = None):
         # TODO: Check if package already exists
 
-        if any([path, giturl]) and not all([path, giturl]):
+        if not any([path, giturl]) or all([path, giturl]):
```

Comment:
should be `if not any([path, giturl]) and not all([path, giturl])`

we want path or giturl to be there but not both of them

Reply:
we need the condition to be the opposite, if it enteres the if it'll raise not continue

thanks @abom

Reply:
i think it is correct
raise error if  not any of path / giturl passed or if both of them are passed